docs/spec.md

```diff
@@ -16,16 +37,15 @@ size 12345
 (ending \n)
 ```
 
-The pointer file should be small (less than 200 bytes), and consist of only
-ASCII characters.  Libraries that generate this should write the file
-identically, so that different implementations write consistent pointers that
-translate to the same Git blob OID.  This means:
+For testing compliance of any tool generating its own pointer files, the
+reference is this official Git LFS tool:
+
+NOTE: exact pointer command behavior TBD!
 
-* Use properties "version", "oid", and "size" in that order.
-* Separate the property from its value with a single space.
-* Oid has a "sha256:" prefix.  No other hashing methods are currently supported
-for Git LFS oids.
-* Size is in bytes.
+* Run `git lfs pointer` to generate a pointer file for the given local file.
+* Run `git lfs pointer` to compare the blob OID of the generated pointer files.
+* Tools that parse and regenerate pointer files MUST preserve keys that they
+don't know or care about.
 
 Note: Earlier versions only contained the OID, with a `# comment` above it.
```

Comment:
This wasn't touched by the current PR but including this part about earlier versions and some out-of-date code for parsing older pointer files has been bothering me :stuck_out_tongue_winking_eye: 

Any reason not to :fire: this section through current line 66?